presto-main/src/main/java/io/prestosql/cost/LocalCostEstimate.java

```diff
@@ -95,6 +105,27 @@ public String toString()
                 .toString();
     }
 
+    @Override
+    public boolean equals(Object o)
```

Comment:
This is needed so that `PlanCostEstimate` can have proper `equals` and `hashCode` on every field it has (and not identity comparison). It's used in tests only I think